addon-test-support/setup-global-a11y-hooks.ts

```diff
@@ -0,0 +1,40 @@
+import { _registerHook, getContext } from '@ember/test-helpers';
+import { InvocationStrategy } from './invocation-strategies';
+import { getRunOptions, setupRunOptions } from './run-options';
+import { HookUnregister } from '@ember/test-helpers';
+import a11yAudit from './audit';
+
+let _unregisterHooks: HookUnregister[] = [];
+
+setupRunOptions();
+
+/**
+ *
+ * @param shouldAudit Invocation strategy function that determines whether to run the audit helper or not.
+ * @param audit Optional audit function used to run the audit. Allows for providing either a11yAudit, a11yAuditIf,
+ *              or custom audit implementation.
+ */
+export function setupGlobalA11yHooks(
+  shouldAudit: InvocationStrategy,
+  audit: (...args: any[]) => PromiseLike<void> = a11yAudit
+) {
+  ['visit', 'click'].forEach((helperName) => {
+    let hook = _registerHook(helperName, 'end', async () => {
+      if (shouldAudit()) {
```

Comment:
I think shouldAudit should receive the helper name / label, so you could implement a custom per-helper system...

Reply:
I like. Adding.